src/main/java/com/box/sdk/BoxItem.java

```diff
@@ -172,12 +173,15 @@ protected void removeWatermark(URLTemplate itemUrl) {
      * <p>This method is a convenience method for manually creating a new shared link and applying it to this item with
      * {@link Info#setSharedLink}. You may want to create the shared link manually so that it can be updated along with
      * other changes to the item's info in a single network request, giving a boost to performance.</p>
+     * @deprecated
+     * Use dedicated methods on subclases to create BoxSharedLink
```

Comment:
Should these two lines be combined into one to match other usages?
Should this be moved to appear right after the `@return` comment on line 182, below?
Fixed a typo and mentioned the method by name.

Reply:
```suggestion
     * @deprecated  Use dedicated <code>createSharedLink(BoxSharedLinkRequest)</code> methods on subclasses to create BoxSharedLink
```